composer.json

```diff
@@ -20,10 +20,9 @@
         }
     },
     "require": {
-        "php": "^7.3",
+        "php": "^7.4",
```

Comment:
You'll want to update this in the pantheon.yml file and roll a new version of the docker instance circle uses to be fully ready to go.

Reply:
We have a working docker instance, but it includes screener. What do you think about just maintaining that one and not worrying about a separate, screener-free image?

Reply:
I'm 100% fine with that.